boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.yaml

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
File name does not need '_cm33' suffix.
Other boards add this suffix because they are multi-core and have asymmetric architecture, e.g. M4+M0.

Reply:
This was added to have the flexibility to add zephyr support for the DSP core at a later time.